### PR TITLE
add consumer group data collector

### PR DESCRIPTION
**Pull request by zmstone**
require brod 2.1.8 (which is not published to hex yet).

### CODE REVIEW

**Comment by id** [comment]
2.1.8 is published.

File: mix.lock [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "gettext": {:hex, :gettext, "0.11.0", "80c1dd42d270482418fa158ec5ba073d2980e3718bacad86f3d4ad71d5667679", [:mix], []},
   "guardian": {:hex, :guardian, "0.12.0", "ab1f0a1ab0cd8f4f9c8cca6e28d61136ca682684cf0f82e55a50e8061be7575a", [:mix], [{:uuid, ">=1.1.1", [hex: :uuid, optional: false]}, {:poison, ">= 1.3.0", [hex: :poison, optional: false]}, {:plug, "~> 1.0", [hex: :plug, optional: false]}, {:jose, "~> 1.6", [hex: :jose, optional: false]}]},
   "jose": {:hex, :jose, "1.7.5", "f19f890f7aaa1261230af691e2f1e0fc60bf44bce7d34c86022d0efa8875628a", [:mix, :rebar], [{:base64url, "~> 0.0.1", [hex: :base64url, optional: false]}]},
-  "kafka_protocol": {:hex, :kafka_protocol, "0.3.2", "36ec836b0c7bb2100985c6807da345145241e3e7a36bd3697ece7b1011bafcaf", [:make, :rebar3], []},
+  "kafka_protocol": {:hex, :kafka_protocol, "0.4.0", "2510b1696142f6e5db01e4b12d4e0c7f14b2c7aef5c2d4ed7b847d485e20e78e", [:make, :rebar3], []},
```
Comment (zmstone) [inline mix.lock:16]: why is it still 0.4 ?

File: lib/kastlex/cg_status_collector.ex [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  defp get_type(group_id) when is_binary(group_id) do
+    :group
+  end
+  defp get_type({group_id, _topic, _partition}) when is_binary(group_id) do
+    :offset
+  end
+
+  defp update_ets(key, _value = []) do
+    :ets.delete(@table, key)
```
Comment (zmstone) [inline lib/kastlex/cg_status_collector.ex:63]: maybe mark it as deleted.

**Comment by zmstone** [comment]
why am i commenting on my own pr.

**Pull request closed without merging**